datasafe-rest-impl/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM openjdk:8-jre-alpine
-ARG JAR_FILE=datasafe-rest-${project.version}.jar
-ENV APP_HOME /usr/app
+ARG JAR_FILE
 EXPOSE 8080
-COPY target/$JAR_FILE $APP_HOME/
+ENV APP_HOME /usr/app
+COPY target/${JAR_FILE} $APP_HOME/app.jar
```

Comment:
Why not preserving jar version, since it is available it makes sense to keep it
app.jar is less informative than datasafe-rest-0.0.5.jar

Reply:
ok